Content/Library/docsTool/templates/master.fs

```diff
@@ -0,0 +1,169 @@
+module Master
+
+open System
+open Fable.React
+open Fable.React.Props
+open DocsTool
+
+type MasterTemplateConfig = {
+    SiteBaseUrl : Uri
+    GitHubRepoUrl : Uri
+    ProjectName : string
+    ReleaseVersion : string
+    ReleaseDate : DateTimeOffset
+}
+
+let renderFooter (cfg : MasterTemplateConfig) (pageSource : string option) =
+    footer [Class "footer font-small m-0 py-4 bg-dark"] [
+        div [Class "container"] [
+            div [Class "row"] [
+                div [Class "col-12 col-md-4 mb-4 mb-md-0"] [
```

Comment:
these link-section divs feel like they could be a component as well, ie pull out the repo-local `a`-generation to a component, then the specific list-group-item formatting, then the overall header-with-list-of-items thing, etc.